tlslite/utils/cryptomath.py

```diff
@@ -126,6 +126,12 @@ def bytesToNumber(b, endian="big"):
 
     By default assumes big-endian encoding of the number.
     """
+    # if string is empty, consider it to be representation of zero
+    # while it may be a bit unorthodox, it is the inverse of numberToByteArray
+    # with default parameters
```

Comment:
Is this interpretation of empty strings generally useful, or only specific part of tlslite-ng relies on that?
If the former it might be good to mention it in the function documentation.  If the latter, perhaps it might be good to check that in the caller.